### ISSUE TITLE
Building high-res T1 T2 template with antsMultivariateTemplateConstruction.sh

### ISSUE THREAD

**Issue by Jinglei-Lv** [ANTsX/ANTs#1138]
Dear ANTs experts,
I was trying to build up the the multi-modal template of T1 and T2 with  antsMultivariateTemplateConstruction.sh, but end up with a half optimized template as shown in the attached picture. What could be the problem?  Specifically, the T1 and T2 data from 55 HCP subjects were used with the resolution of 0.7x0.7x0.7mm. I have set the iteration to 10.  

Thank you a lot.
Best,
Jinglei

![Screenshot from 2021-01-27 14-55-19](https://user-images.githubusercontent.com/16005941/105940607-cd9b9180-60af-11eb-96fe-14b54991fcbb.png)

**Comment by cookpa** [comment]
What's your ANTs version? Run `antsRegistration --version` to find out

**Comment by Jinglei-Lv** [comment]
Dear Philip
The ANTs version is as follow:
ANTs Version: 2.3.1.dev106-ga025d
Thanks.

best,
Jinglei

**Comment by cookpa** [comment]
Thanks, I think you have a bug in ANTS, the old registration program used in that script. If you get the latest release, this is fixed. Alternatively you could try `antsMultivariateTemplateConstruction2.sh`, which is slower, but has a newer registration with more options.

**Comment by Jinglei-Lv** [comment]
Hi Philip, I will try both. Thanks.

**Comment by gdevenyi** [comment]
You may interested in this:
https://www.nature.com/articles/s41597-020-00644-6?sf237643362=1
https://springernature.figshare.com/articles/dataset/Metadata_record_for_A_high-resolution_in-vivo_magnetic_resonance_imaging_atlas_of_the_human_hypothalamic_region/12888560

**Comment by cookpa** [comment]
BTW the multivariate template construction requires that both modalities be aligned, ie each subject's T1w and T2w image need to be in the same physical space. I don't think the HCP bandwidth-matched the T1w and T2w, so there might be a slight distortion difference. Also, if there is motion, you'd need to correct that before running the template script.

**Comment by gdevenyi** [comment]
HCP did an extensive amount of distortion correction and standardization to ensure alignment of their multispectral acquisitions.

**Comment by cookpa** [comment]
Yes good point @gdevenyi, I forget people might be using the preprocessed rather than the raw data

**Comment by Jinglei-Lv** [comment]
Thank you both @cookpa @gdevenyi . I'm using the preprocessed and well aligned T1 and T2.

**Comment by Jinglei-Lv** [comment]
> You may interested in this:
> https://www.nature.com/articles/s41597-020-00644-6?sf237643362=1
> https://springernature.figshare.com/articles/dataset/Metadata_record_for_A_high-resolution_in-vivo_magnetic_resonance_imaging_atlas_of_the_human_hypothalamic_region/12888560



> You may interested in this:
> https://www.nature.com/articles/s41597-020-00644-6?sf237643362=1
> https://springernature.figshare.com/articles/dataset/Metadata_record_for_A_high-resolution_in-vivo_magnetic_resonance_imaging_atlas_of_the_human_hypothalamic_region/12888560

It looks amazing. Thank you. But do you think I can directly use the stripped brains generated from the minimal preprocessing pipeline from HCP?

**Comment by gdevenyi** [comment]
I don't know, you'll have to experiment with what you want to do.
